keyboards/keychron/k8/rgb/keymaps/default/rules.mk

```diff
@@ -0,0 +1,14 @@
+# https://beta.docs.qmk.fm/developing-qmk/qmk-reference/getting_started_make_guide#rules-mk-options
+
+# AUTO_SHIFT_ENABLE = yes
+# MOUSEKEY_ENABLE = yes
+EXTRAKEY_ENABLE = yes    # Audio control and System control
```

Comment:
why is this here?

Reply:
Because I didn't look through the keymap files ;) But it's removed, as it's in the base info.json file